[move_group-1]   what():  expected [string_array] got [string]

### Affected ROS2 Driver version(s)

Humble

### Used ROS distribution.

Humble

### Which combination of platform is the ROS driver running on.

Ubuntu Linux with standard kernel

### How is the UR ROS2 Driver installed.

From binary packets

### Which robot platform is the driver connected to.

URSim in docker

### Robot SW / URSim version(s)

5.14.5

### How is the ROS driver used.

Through the robot teach pendant using External Control URCap

### Issue details

# Summary
When i do launch the ur_moveit.launch.py i got the the following issue:
[move_group-1] terminate called after throwing an instance of 'rclcpp::ParameterTypeException'
[move_group-1]   what():  expected [string_array] got [string]

After that rviz does start but i cant plan a path.

## Steps to Reproduce
### Setup
- install ros2 humble.
- install moveit2 from https://moveit.picknik.ai/main/doc/tutorials/getting_started/getting_started.html
- install Universal Robot ROS2 Driver `sudo apt-get install ros-humble-ur`
  
### Launch
- `ros2 run ur_client_library start_ursim.sh -m ur3e`
- `ros2 launch ur_robot_driver ur_control.launch.py ur_type:=ur3e robot_ip:=192.168.56.101 launch_rviz:=false`
- `ros2 launch ur_moveit_config ur_moveit.launch.py ur_type:=ur3e`


## Expected Behavior
Start rviz and with working moveit planning

## Actual Behavior
rviz does start but you can not plan or execute trajectory's. The Robot is visible and does move when controlling in the sim. 


### Relevant log output

```shell
[INFO] [launch]: All log files can be found below /home/harrrlan/.ros/log/2024-01-04-12-12-31-862815-harrrlan-Surface-Book-2-38360
[INFO] [launch]: Default logging verbosity is set to INFO
[INFO] [move_group-1]: process started with pid [38369]
[INFO] [rviz2-2]: process started with pid [38371]
[INFO] [servo_node_main-3]: process started with pid [38373]
[rviz2-2] Warning: Ignoring XDG_SESSION_TYPE=wayland on Gnome. Use QT_QPA_PLATFORM=wayland to run on Wayland anyway.
[servo_node_main-3] [WARN] [1704366753.395000186] [servo_node]: Intra-process communication is disabled, consider enabling it by adding: 
[servo_node_main-3] extra_arguments=[{'use_intra_process_comms' : True}]
[servo_node_main-3] to the Servo composable node in the launch file
[servo_node_main-3] [WARN] [1704366753.395175995] [servo_node]: Realtime kernel is recommended for better performance.
[servo_node_main-3] [INFO] [1704366753.404174349] [servo_node.moveit.RDFLoader]: Loaded robot model in 0.00758735 seconds
[servo_node_main-3] [INFO] [1704366753.404231430] [servo_node.moveit.robot_model]: Loading robot model 'ur'...
[servo_node_main-3] [INFO] [1704366753.404255086] [servo_node.moveit.robot_model]: No root/virtual joint specified in SRDF. Assuming fixed joint
[move_group-1] [WARN] [1704366753.406611712] [move_group]: Falling back to using the the move_group node namespace (deprecated behavior).
[move_group-1] [INFO] [1704366753.413191955] [move_group.moveit.RDFLoader]: Loaded robot model in 0.00624718 seconds
[move_group-1] [INFO] [1704366753.413235678] [move_group.moveit.robot_model]: Loading robot model 'ur'...
[move_group-1] [INFO] [1704366753.413256049] [move_group.moveit.robot_model]: No root/virtual joint specified in SRDF. Assuming fixed joint
[servo_node_main-3] [WARN] [1704366753.445443495] [servo_node.moveit.robot_model_loader]: No kinematics plugins defined. Fill and load kinematics.yaml!
[move_group-1] [INFO] [1704366753.449724555] [moveit_kdl_kinematics_plugin.kdl_kinematics_plugin]: Joint weights for group 'ur_manipulator': 1 1 1 1 1 1
[servo_node_main-3] [INFO] [1704366753.504629044] [servo_node.moveit.current_state_monitor]: Listening to joint states on topic '/joint_states'
[servo_node_main-3] [INFO] [1704366753.509493362] [servo_node.moveit.planning_scene_monitor]: Listening to '/attached_collision_object' for attached collision objects
[servo_node_main-3] [INFO] [1704366753.509521244] [servo_node.moveit.planning_scene_monitor]: Starting planning scene monitor
[servo_node_main-3] [INFO] [1704366753.510760520] [servo_node.moveit.planning_scene_monitor]: Listening to '/planning_scene'
[servo_node_main-3] [INFO] [1704366753.511094834] [servo_node.moveit.planning_scene_monitor]: Publishing maintained planning scene on '/servo_node/publish_planning_scene'
[move_group-1] [INFO] [1704366753.511703982] [move_group.moveit.planning_scene_monitor]: Publishing maintained planning scene on 'monitored_planning_scene'
[move_group-1] [INFO] [1704366753.511850922] [move_group.moveit.moveit_cpp]: Listening to 'joint_states' for joint states
[move_group-1] [INFO] [1704366753.512579611] [move_group.moveit.current_state_monitor]: Listening to joint states on topic 'joint_states'
[move_group-1] [INFO] [1704366753.512977848] [move_group.moveit.planning_scene_monitor]: Listening to '/attached_collision_object' for attached collision objects
[move_group-1] [INFO] [1704366753.512997222] [move_group.moveit.planning_scene_monitor]: Starting planning scene monitor
[move_group-1] [INFO] [1704366753.513279265] [move_group.moveit.planning_scene_monitor]: Listening to '/planning_scene'
[move_group-1] [INFO] [1704366753.513293786] [move_group.moveit.planning_scene_monitor]: Starting world geometry update monitor for collision objects, attached objects, octomap updates.
[move_group-1] [INFO] [1704366753.513627346] [move_group.moveit.planning_scene_monitor]: Listening to 'collision_object'
[move_group-1] [INFO] [1704366753.513954915] [move_group.moveit.planning_scene_monitor]: Listening to 'planning_scene_world' for planning scene world geometry
[move_group-1] [WARN] [1704366753.516827818] [move_group.moveit.occupancy_map_monitor]: Resolution not specified for Octomap. Assuming resolution = 0.1 instead
[move_group-1] [ERROR] [1704366753.516852665] [move_group.moveit.occupancy_map_monitor]: No 3D sensor plugin(s) defined for octomap updates
[move_group-1] terminate called after throwing an instance of 'rclcpp::ParameterTypeException'
[move_group-1]   what():  expected [string_array] got [string]
[move_group-1] Stack trace (most recent call last):
[move_group-1] #17   Object "", at 0xffffffffffffffff, in 
[move_group-1] #16   Object "/home/harrrlan/ws_moveit/install/moveit_ros_move_group/lib/moveit_ros_move_group/move_group", at 0x63487e87e094, in _start
[move_group-1] #15   Source "../csu/libc-start.c", line 392, in __libc_start_main_impl [0x72f53a629e3f]
[move_group-1] #14   Source "../sysdeps/nptl/libc_start_call_main.h", line 58, in __libc_start_call_main [0x72f53a629d8f]
[move_group-1] #13   Object "/home/harrrlan/ws_moveit/install/moveit_ros_move_group/lib/moveit_ros_move_group/move_group", at 0x63487e87d21e, in main
[move_group-1] #12   Object "/home/harrrlan/ws_moveit/install/moveit_ros_planning/lib/libmoveit_cpp.so.2.8.0", at 0x72f53b31a240, in moveit_cpp::MoveItCpp::MoveItCpp(std::shared_ptr<rclcpp::Node> const&, moveit_cpp::MoveItCpp::Options const&)
[move_group-1] #11   Object "/home/harrrlan/ws_moveit/install/moveit_ros_planning/lib/libmoveit_cpp.so.2.8.0", at 0x72f53b31927b, in moveit_cpp::MoveItCpp::loadPlanningPipelines(moveit_cpp::MoveItCpp::PlanningPipelineOptions const&)
[move_group-1] #10   Object "/home/harrrlan/ws_moveit/install/moveit_ros_planning/lib/libmoveit_planning_pipeline_interfaces.so.2.8.0", at 0x72f53a997b06, in moveit::planning_pipeline_interfaces::createPlanningPipelineMap(std::vector<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::allocator<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > > const&, std::shared_ptr<moveit::core::RobotModel const> const&, std::shared_ptr<rclcpp::Node> const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)
[move_group-1] #9    Object "/home/harrrlan/ws_moveit/install/moveit_ros_planning/lib/libmoveit_planning_pipeline.so.2.8.0", at 0x72f53a593e56, in planning_pipeline::PlanningPipeline::PlanningPipeline(std::shared_ptr<moveit::core::RobotModel const> const&, std::shared_ptr<rclcpp::Node> const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)
[move_group-1] #8    Object "/home/harrrlan/ws_moveit/install/moveit_ros_planning/lib/libmoveit_planning_pipeline.so.2.8.0", at 0x72f53a5d7c5b, in planning_pipeline_parameters::ParamListener::declare_params()
[move_group-1] #7    Object "/opt/ros/humble/lib/librclcpp.so", at 0x72f53aeca863, in 
[move_group-1] #6    Object "/usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.30", at 0x72f53aaae4d7, in __cxa_throw
[move_group-1] #5    Object "/usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.30", at 0x72f53aaae276, in std::terminate()
[move_group-1] #4    Object "/usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.30", at 0x72f53aaae20b, in 
[move_group-1] #3    Object "/usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.30", at 0x72f53aaa2b9d, in 
[move_group-1] #2    Source "./stdlib/abort.c", line 79, in abort [0x72f53a6287f2]
[move_group-1] #1    Source "../sysdeps/posix/raise.c", line 26, in raise [0x72f53a642475]
[move_group-1] #0  | Source "./nptl/pthread_kill.c", line 89, in __pthread_kill_internal
[move_group-1]     | Source "./nptl/pthread_kill.c", line 78, in __pthread_kill_implementation
[move_group-1]       Source "./nptl/pthread_kill.c", line 44, in __pthread_kill [0x72f53a6969fc]
[move_group-1] Aborted (Signal sent by tkill() 38369 1000)
[servo_node_main-3] Warning: class_loader.ClassLoader: SEVERE WARNING!!! Attempting to unload library while objects created by this loader exist in the heap! You should delete your objects before attempting to unload the library or destroying the ClassLoader. The library will NOT be unloaded.
[servo_node_main-3]          at line 127 in ./src/class_loader.cpp
[servo_node_main-3] [INFO] [1704366753.670332772] [servo_node.moveit.CollisionMonitor]: Collision monitor started
[servo_node_main-3] [INFO] [1704366753.670358200] [servo_node.moveit.servo]: Servo initialized successfully
[ERROR] [move_group-1]: process has died [pid 38369, exit code -6, cmd '/home/harrrlan/ws_moveit/install/moveit_ros_move_group/lib/moveit_ros_move_group/move_group --ros-args --params-file /tmp/launch_params_fcjbq2oa --params-file /tmp/launch_params_b7ov86ml --params-file /opt/ros/humble/share/ur_moveit_config/config/kinematics.yaml --params-file /tmp/launch_params_izxjx8cg --params-file /tmp/launch_params_ngaujua2 --params-file /tmp/launch_params_5isobd96 --params-file /tmp/launch_params_0rrebd1s --params-file /tmp/launch_params__j2g79dn --params-file /tmp/launch_params_s4icfg4x'].
[rviz2-2] [INFO] [1704366753.812529949] [rviz2]: Stereo is NOT SUPPORTED
[rviz2-2] [INFO] [1704366753.812621476] [rviz2]: OpenGl version: 4.6 (GLSL 4.6)
[rviz2-2] [INFO] [1704366753.841392040] [rviz2]: Stereo is NOT SUPPORTED
[rviz2-2] Warning: class_loader.impl: SEVERE WARNING!!! A namespace collision has occurred with plugin factory for class rviz_default_plugins::displays::InteractiveMarkerDisplay. New factory will OVERWRITE existing one. This situation occurs when libraries containing plugins are directly linked against an executable (the one running right now generating this message). Please separate plugins out into their own library or just don't link against the library and use either class_loader::ClassLoader/MultiLibraryClassLoader to open.
[rviz2-2]          at line 253 in /opt/ros/humble/include/class_loader/class_loader/class_loader_core.hpp
[rviz2-2] [WARN] [1704366754.030851635] [rcl.logging_rosout]: Publisher already registered for provided node name. If this is due to multiple nodes with the same name then all logs for that logger name will go out over the existing publisher. As soon as any node with that name is destructed it will unregister the publisher, preventing any further logs for that name from being published on the rosout topic.
[rviz2-2] [ERROR] [1704366757.070810305] [rviz2_moveit.motion_planning_frame]: Action server: /recognize_objects not available
[rviz2-2] [INFO] [1704366757.159029266] [rviz2_moveit.motion_planning_frame]: MoveGroup namespace changed: / -> . Reloading params.
[rviz2-2] [INFO] [1704366757.263702360] [rviz2_moveit.RDFLoader]: Loaded robot model in 0.007868 seconds
[rviz2-2] [INFO] [1704366757.263797608] [rviz2_moveit.robot_model]: Loading robot model 'ur'...
[rviz2-2] [INFO] [1704366757.263850101] [rviz2_moveit.robot_model]: No root/virtual joint specified in SRDF. Assuming fixed joint
[rviz2-2] [INFO] [1704366757.312762665] [moveit_kdl_kinematics_plugin.kdl_kinematics_plugin]: Joint weights for group 'ur_manipulator': 1 1 1 1 1 1
[rviz2-2] [INFO] [1704366757.447795605] [rviz2_moveit.planning_scene_monitor]: Starting planning scene monitor
[rviz2-2] [INFO] [1704366757.448848748] [rviz2_moveit.planning_scene_monitor]: Listening to '/monitored_planning_scene'
[rviz2-2] [INFO] [1704366757.794691060] [interactive_marker_display_101519103898192]: Connected on namespace: /rviz_moveit_motion_planning_display/robot_interaction_interactive_marker_topic
[rviz2-2] [INFO] [1704366757.801463012] [rviz2_moveit.motion_planning_frame]: group ur_manipulator
[rviz2-2] [INFO] [1704366757.801487852] [rviz2_moveit.motion_planning_frame]: Constructing new MoveGroup connection for group 'ur_manipulator' in namespace ''
[rviz2-2] [INFO] [1704366757.814224289] [interactive_marker_display_101519103898192]: Sending request for interactive markers
[rviz2-2] [INFO] [1704366757.845203739] [interactive_marker_display_101519103898192]: Service response received for initialization
[rviz2-2] [INFO] [1704366817.836302434] [rviz2_moveit.move_group_interface]: Ready to take commands for planning group ur_manipulator.
```


### Accept Public visibility

- [X] I agree to make this context public

Also encountered this. Should be fixed by #906.

I build moveit2 with debug enabled, the error log is:
```
❯ ros2 launch ur_moveit_config ur_moveit.launch.py ur_type:=ur5e                                                                                                                           ─╯
[INFO] [launch]: All log files can be found below /home/ortho/.ros/log/2024-01-11-11-08-45-214340-ortho-pc-844748
[INFO] [launch]: Default logging verbosity is set to INFO
[INFO] [move_group-1]: process started with pid [844762]
[INFO] [rviz2-2]: process started with pid [844764]
[INFO] [servo_node-3]: process started with pid [844766]
[servo_node-3] [WARN] [1704942525.747747542] [servo_node]: Intra-process communication is disabled, consider enabling it by adding: 
[servo_node-3] extra_arguments=[{'use_intra_process_comms' : True}]
[servo_node-3] to the Servo composable node in the launch file
[servo_node-3] [WARN] [1704942525.747841954] [servo_node]: Realtime kernel is recommended for better performance.
[servo_node-3] [INFO] [1704942525.751180506] [servo_node.moveit.RDFLoader]: Loaded robot model in 0.00214597 seconds
[servo_node-3] [INFO] [1704942525.751441315] [servo_node.moveit.robot_model]: Loading robot model 'ur'...
[servo_node-3] [INFO] [1704942525.751491313] [servo_node.moveit.robot_model]: No root/virtual joint specified in SRDF. Assuming fixed joint
[move_group-1] [WARN] [1704942525.751831776] [move_group]: Falling back to using the the move_group node namespace (deprecated behavior).
[move_group-1] [INFO] [1704942525.753825235] [move_group.moveit.RDFLoader]: Loaded robot model in 0.00167837 seconds
[move_group-1] [INFO] [1704942525.754030087] [move_group.moveit.robot_model]: Loading robot model 'ur'...
[move_group-1] [INFO] [1704942525.754067590] [move_group.moveit.robot_model]: No root/virtual joint specified in SRDF. Assuming fixed joint
[servo_node-3] [INFO] [1704942525.775621923] [servo_node.moveit.kdl_kinematics_plugin]: Joint weights for group 'ur_manipulator': 1 1 1 1 1 1
[move_group-1] [INFO] [1704942525.775636842] [move_group.moveit.kdl_kinematics_plugin]: Joint weights for group 'ur_manipulator': 1 1 1 1 1 1
[servo_node-3] [INFO] [1704942525.903823835] [servo_node.moveit.current_state_monitor]: Listening to joint states on topic '/joint_states'
[servo_node-3] [INFO] [1704942525.906732488] [servo_node.moveit.planning_scene_monitor]: Listening to '/attached_collision_object' for attached collision objects
[servo_node-3] [INFO] [1704942525.906760355] [servo_node.moveit.planning_scene_monitor]: Starting planning scene monitor
[servo_node-3] [INFO] [1704942525.907840971] [servo_node.moveit.planning_scene_monitor]: Listening to '/planning_scene'
[servo_node-3] [INFO] [1704942525.907873668] [servo_node.moveit.planning_scene_monitor]: Starting world geometry update monitor for collision objects, attached objects, octomap updates.
[servo_node-3] [INFO] [1704942525.908566091] [servo_node.moveit.planning_scene_monitor]: Listening to 'collision_object'
[servo_node-3] [INFO] [1704942525.909271252] [servo_node.moveit.planning_scene_monitor]: Listening to 'planning_scene_world' for planning scene world geometry
[servo_node-3] [INFO] [1704942525.910622831] [servo_node.moveit.planning_scene_monitor]: Publishing maintained planning scene on '/servo_node/publish_planning_scene'
[servo_node-3] Warning: class_loader.ClassLoader: SEVERE WARNING!!! Attempting to unload library while objects created by this loader exist in the heap! You should delete your objects before attempting to unload the library or destroying the ClassLoader. The library will NOT be unloaded.
[servo_node-3]          at line 127 in ./src/class_loader.cpp
[move_group-1] [INFO] [1704942525.913470136] [move_group.moveit.planning_scene_monitor]: Publishing maintained planning scene on 'monitored_planning_scene'
[servo_node-3] [INFO] [1704942525.913874457] [servo_node.moveit.CollisionMonitor]: Collision monitor started
[servo_node-3] [INFO] [1704942525.913896205] [servo_node.moveit.servo]: Servo initialized successfully
[move_group-1] [INFO] [1704942525.913896045] [move_group.moveit.moveit_cpp]: Listening to 'joint_states' for joint states
[move_group-1] [INFO] [1704942525.914818825] [move_group.moveit.current_state_monitor]: Listening to joint states on topic 'joint_states'
[move_group-1] [INFO] [1704942525.915464295] [move_group.moveit.planning_scene_monitor]: Listening to '/attached_collision_object' for attached collision objects
[move_group-1] [INFO] [1704942525.915483117] [move_group.moveit.planning_scene_monitor]: Starting planning scene monitor
[move_group-1] [INFO] [1704942525.916252357] [move_group.moveit.planning_scene_monitor]: Listening to '/planning_scene'
[move_group-1] [INFO] [1704942525.916269155] [move_group.moveit.planning_scene_monitor]: Starting world geometry update monitor for collision objects, attached objects, octomap updates.
[move_group-1] [INFO] [1704942525.916898126] [move_group.moveit.planning_scene_monitor]: Listening to 'collision_object'
[move_group-1] [INFO] [1704942525.917771056] [move_group.moveit.planning_scene_monitor]: Listening to 'planning_scene_world' for planning scene world geometry
[servo_node-3] [INFO] [1704942525.917780353] [servo_node]: Waiting to receive robot state update.
[move_group-1] [WARN] [1704942525.918324974] [move_group.moveit.occupancy_map_monitor]: Resolution not specified for Octomap. Assuming resolution = 0.1 instead
[move_group-1] [ERROR] [1704942525.918340705] [move_group.moveit.occupancy_map_monitor]: No 3D sensor plugin(s) defined for octomap updates
[move_group-1] terminate called after throwing an instance of 'rclcpp::ParameterTypeException'
[move_group-1]   what():  expected [string_array] got [string]
[move_group-1] Stack trace (most recent call last):
[move_group-1] #31   Source "/home/ortho/ws_moveit/src/moveit2/moveit_ros/move_group/src/move_group.cpp", line 287, in main [0x555e5f67acb8]
[move_group-1]         284:   }
[move_group-1]         285: 
[move_group-1]         286:   // Initialize MoveItCpp
[move_group-1]       > 287:   const auto moveit_cpp = std::make_shared<moveit_cpp::MoveItCpp>(nh, moveit_cpp_options);
[move_group-1]         288:   const auto planning_scene_monitor = moveit_cpp->getPlanningSceneMonitorNonConst();
[move_group-1]         289: 
[move_group-1]         290:   if (planning_scene_monitor->getPlanningScene())
[move_group-1] #30   Source "/usr/include/c++/11/bits/shared_ptr.h", line 879, in make_shared<moveit_cpp::MoveItCpp, std::shared_ptr<rclcpp::Node>&, moveit_cpp::MoveItCpp::Options&> [0x555e5f685383]
[move_group-1]         876:     {
[move_group-1]         877:       typedef typename std::remove_cv<_Tp>::type _Tp_nc;
[move_group-1]         878:       return std::allocate_shared<_Tp>(std::allocator<_Tp_nc>(),
[move_group-1]       > 879: 				       std::forward<_Args>(__args)...);
[move_group-1]         880:     }
[move_group-1]         881: 
[move_group-1]         882:   /// std::hash specialization for shared_ptr.
[move_group-1] #29   Source "/usr/include/c++/11/bits/shared_ptr.h", line 863, in allocate_shared<moveit_cpp::MoveItCpp, std::allocator<moveit_cpp::MoveItCpp>, std::shared_ptr<rclcpp::Node>&, moveit_cpp::MoveItCpp::Options&> [0x555e5f689f82]
[move_group-1]         860:       static_assert(!is_array<_Tp>::value, "make_shared<T[]> not supported");
[move_group-1]         861: 
[move_group-1]         862:       return shared_ptr<_Tp>(_Sp_alloc_shared_tag<_Alloc>{__a},
[move_group-1]       > 863: 			     std::forward<_Args>(__args)...);
[move_group-1]         864:     }
[move_group-1]         865: 
[move_group-1]         866:   /**
[move_group-1] #28   Source "/usr/include/c++/11/bits/shared_ptr.h", line 409, in shared_ptr<std::allocator<moveit_cpp::MoveItCpp>, std::shared_ptr<rclcpp::Node>&, moveit_cpp::MoveItCpp::Options&> [0x555e5f68ef9c]
[move_group-1]         406:       // This constructor is non-standard, it is used by allocate_shared.
[move_group-1]         407:       template<typename _Alloc, typename... _Args>
[move_group-1]         408: 	shared_ptr(_Sp_alloc_shared_tag<_Alloc> __tag, _Args&&... __args)
[move_group-1]       > 409: 	: __shared_ptr<_Tp>(__tag, std::forward<_Args>(__args)...)
[move_group-1]         410: 	{ }
[move_group-1]         411: 
[move_group-1]         412:       template<typename _Yp, typename _Alloc, typename... _Args>
[move_group-1] #27   Source "/usr/include/c++/11/bits/shared_ptr_base.h", line 1342, in __shared_ptr<std::allocator<moveit_cpp::MoveItCpp>, std::shared_ptr<rclcpp::Node>&, moveit_cpp::MoveItCpp::Options&> [0x555e5f694a1d]
[move_group-1]        1339:       // This constructor is non-standard, it is used by allocate_shared.
[move_group-1]        1340:       template<typename _Alloc, typename... _Args>
[move_group-1]        1341: 	__shared_ptr(_Sp_alloc_shared_tag<_Alloc> __tag, _Args&&... __args)
[move_group-1]       >1342: 	: _M_ptr(), _M_refcount(_M_ptr, __tag, std::forward<_Args>(__args)...)
[move_group-1]        1343: 	{ _M_enable_shared_from_this_with(_M_ptr); }
[move_group-1]        1344: 
[move_group-1]        1345:       template<typename _Tp1, _Lock_policy _Lp1, typename _Alloc,
[move_group-1] #26   Source "/usr/include/c++/11/bits/shared_ptr_base.h", line 650, in __shared_count<moveit_cpp::MoveItCpp, std::allocator<moveit_cpp::MoveItCpp>, std::shared_ptr<rclcpp::Node>&, moveit_cpp::MoveItCpp::Options&> [0x555e5f698ae0]
[move_group-1]         647: 	  typename _Sp_cp_type::__allocator_type __a2(__a._M_a);
[move_group-1]         648: 	  auto __guard = std::__allocate_guarded(__a2);
[move_group-1]         649: 	  _Sp_cp_type* __mem = __guard.get();
[move_group-1]       > 650: 	  auto __pi = ::new (__mem)
[move_group-1]         651: 	    _Sp_cp_type(__a._M_a, std::forward<_Args>(__args)...);
[move_group-1]         652: 	  __guard = nullptr;
[move_group-1]         653: 	  _M_pi = __pi;
[move_group-1] #25   Source "/usr/include/c++/11/bits/shared_ptr_base.h", line 519, in _Sp_counted_ptr_inplace<std::shared_ptr<rclcpp::Node>&, moveit_cpp::MoveItCpp::Options&> [0x555e5f69caa4]
[move_group-1]         516: 	{
[move_group-1]         517: 	  // _GLIBCXX_RESOLVE_LIB_DEFECTS
[move_group-1]         518: 	  // 2070.  allocate_shared should use allocator_traits<A>::construct
[move_group-1]       > 519: 	  allocator_traits<_Alloc>::construct(__a, _M_ptr(),
[move_group-1]         520: 	      std::forward<_Args>(__args)...); // might throw
[move_group-1]         521: 	}
[move_group-1] #24   Source "/usr/include/c++/11/bits/alloc_traits.h", line 516, in construct<moveit_cpp::MoveItCpp, std::shared_ptr<rclcpp::Node>&, moveit_cpp::MoveItCpp::Options&> [0x555e5f6a0a63]
[move_group-1]         513: 	noexcept(std::is_nothrow_constructible<_Up, _Args...>::value)
[move_group-1]         514: 	{
[move_group-1]         515: #if __cplusplus <= 201703L
[move_group-1]       > 516: 	  __a.construct(__p, std::forward<_Args>(__args)...);
[move_group-1]         517: #else
[move_group-1]         518: 	  std::construct_at(__p, std::forward<_Args>(__args)...);
[move_group-1]         519: #endif
[move_group-1] #23   Source "/usr/include/c++/11/ext/new_allocator.h", line 162, in construct<moveit_cpp::MoveItCpp, std::shared_ptr<rclcpp::Node>&, moveit_cpp::MoveItCpp::Options&> [0x555e5f6a5f2e]
[move_group-1]         159: 	void
[move_group-1]         160: 	construct(_Up* __p, _Args&&... __args)
[move_group-1]         161: 	noexcept(std::is_nothrow_constructible<_Up, _Args...>::value)
[move_group-1]       > 162: 	{ ::new((void *)__p) _Up(std::forward<_Args>(__args)...); }
[move_group-1]         163: 
[move_group-1]         164:       template<typename _Up>
[move_group-1]         165: 	void
[move_group-1] #22   Source "/home/ortho/ws_moveit/src/moveit2/moveit_ros/planning/moveit_cpp/src/moveit_cpp.cpp", line 72, in MoveItCpp [0x7f59c4359c3d]
[move_group-1]          69:   }
[move_group-1]          70: 
[move_group-1]          71:   bool load_planning_pipelines = true;
[move_group-1]       >  72:   if (load_planning_pipelines && !loadPlanningPipelines(options.planning_pipeline_options))
[move_group-1]          73:   {
[move_group-1]          74:     const std::string error = "Failed to load planning pipelines from parameter server";
[move_group-1]          75:     RCLCPP_FATAL_STREAM(logger_, error);
[move_group-1] #21   Source "/home/ortho/ws_moveit/src/moveit2/moveit_ros/planning/moveit_cpp/src/moveit_cpp.cpp", line 130, in loadPlanningPipelines [0x7f59c435ac69]
[move_group-1]         127: {
[move_group-1]         128:   // TODO(henningkayser): Use parent namespace for planning pipeline config lookup
[move_group-1]         129:   planning_pipelines_ =
[move_group-1]       > 130:       moveit::planning_pipeline_interfaces::createPlanningPipelineMap(options.pipeline_names, getRobotModel(), node_);
[move_group-1]         131: 
[move_group-1]         132:   if (planning_pipelines_.empty())
[move_group-1]         133:   {
[move_group-1] #20   Source "/home/ortho/ws_moveit/src/moveit2/moveit_ros/planning/planning_pipeline_interfaces/src/planning_pipeline_interfaces.cpp", line 182, in createPlanningPipelineMap [0x7f59c2fd5e2b]
[move_group-1]         180:     // Create planning pipeline
[move_group-1]         181:     planning_pipeline::PlanningPipelinePtr pipeline = std::make_shared<planning_pipeline::PlanningPipeline>(
[move_group-1]       > 182:         robot_model, node, parameter_namespace + planning_pipeline_name);
[move_group-1]         183: 
[move_group-1]         184:     if (!pipeline)
[move_group-1]         185:     {
[move_group-1] #19   Source "/usr/include/c++/11/bits/shared_ptr.h", line 879, in make_shared<planning_pipeline::PlanningPipeline, const std::shared_ptr<const moveit::core::RobotModel>&, const std::shared_ptr<rclcpp::Node>&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > [0x7f59c2fdccf1]
[move_group-1]         876:     {
[move_group-1]         877:       typedef typename std::remove_cv<_Tp>::type _Tp_nc;
[move_group-1]         878:       return std::allocate_shared<_Tp>(std::allocator<_Tp_nc>(),
[move_group-1]       > 879: 				       std::forward<_Args>(__args)...);
[move_group-1]         880:     }
[move_group-1]         881: 
[move_group-1]         882:   /// std::hash specialization for shared_ptr.
[move_group-1] #18   Source "/usr/include/c++/11/bits/shared_ptr.h", line 863, in allocate_shared<planning_pipeline::PlanningPipeline, std::allocator<planning_pipeline::PlanningPipeline>, const std::shared_ptr<const moveit::core::RobotModel>&, const std::shared_ptr<rclcpp::Node>&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > [0x7f59c2fe0568]
[move_group-1]         860:       static_assert(!is_array<_Tp>::value, "make_shared<T[]> not supported");
[move_group-1]         861: 
[move_group-1]         862:       return shared_ptr<_Tp>(_Sp_alloc_shared_tag<_Alloc>{__a},
[move_group-1]       > 863: 			     std::forward<_Args>(__args)...);
[move_group-1]         864:     }
[move_group-1]         865: 
[move_group-1]         866:   /**
[move_group-1] #17   Source "/usr/include/c++/11/bits/shared_ptr.h", line 409, in shared_ptr<std::allocator<planning_pipeline::PlanningPipeline>, const std::shared_ptr<const moveit::core::RobotModel>&, const std::shared_ptr<rclcpp::Node>&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > [0x7f59c2fe2ffa]
[move_group-1]         406:       // This constructor is non-standard, it is used by allocate_shared.
[move_group-1]         407:       template<typename _Alloc, typename... _Args>
[move_group-1]         408: 	shared_ptr(_Sp_alloc_shared_tag<_Alloc> __tag, _Args&&... __args)
[move_group-1]       > 409: 	: __shared_ptr<_Tp>(__tag, std::forward<_Args>(__args)...)
[move_group-1]         410: 	{ }
[move_group-1]         411: 
[move_group-1]         412:       template<typename _Yp, typename _Alloc, typename... _Args>
[move_group-1] #16   Source "/usr/include/c++/11/bits/shared_ptr_base.h", line 1342, in __shared_ptr<std::allocator<planning_pipeline::PlanningPipeline>, const std::shared_ptr<const moveit::core::RobotModel>&, const std::shared_ptr<rclcpp::Node>&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > [0x7f59c2fe5613]
[move_group-1]        1339:       // This constructor is non-standard, it is used by allocate_shared.
[move_group-1]        1340:       template<typename _Alloc, typename... _Args>
[move_group-1]        1341: 	__shared_ptr(_Sp_alloc_shared_tag<_Alloc> __tag, _Args&&... __args)
[move_group-1]       >1342: 	: _M_ptr(), _M_refcount(_M_ptr, __tag, std::forward<_Args>(__args)...)
[move_group-1]        1343: 	{ _M_enable_shared_from_this_with(_M_ptr); }
[move_group-1]        1344: 
[move_group-1]        1345:       template<typename _Tp1, _Lock_policy _Lp1, typename _Alloc,
[move_group-1] #15   Source "/usr/include/c++/11/bits/shared_ptr_base.h", line 650, in __shared_count<planning_pipeline::PlanningPipeline, std::allocator<planning_pipeline::PlanningPipeline>, const std::shared_ptr<const moveit::core::RobotModel>&, const std::shared_ptr<rclcpp::Node>&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > [0x7f59c2fe6e0d]
[move_group-1]         647: 	  typename _Sp_cp_type::__allocator_type __a2(__a._M_a);
[move_group-1]         648: 	  auto __guard = std::__allocate_guarded(__a2);
[move_group-1]         649: 	  _Sp_cp_type* __mem = __guard.get();
[move_group-1]       > 650: 	  auto __pi = ::new (__mem)
[move_group-1]         651: 	    _Sp_cp_type(__a._M_a, std::forward<_Args>(__args)...);
[move_group-1]         652: 	  __guard = nullptr;
[move_group-1]         653: 	  _M_pi = __pi;
[move_group-1] #14   Source "/usr/include/c++/11/bits/shared_ptr_base.h", line 519, in _Sp_counted_ptr_inplace<const std::shared_ptr<const moveit::core::RobotModel>&, const std::shared_ptr<rclcpp::Node>&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > [0x7f59c2fe7e86]
[move_group-1]         516: 	{
[move_group-1]         517: 	  // _GLIBCXX_RESOLVE_LIB_DEFECTS
[move_group-1]         518: 	  // 2070.  allocate_shared should use allocator_traits<A>::construct
[move_group-1]       > 519: 	  allocator_traits<_Alloc>::construct(__a, _M_ptr(),
[move_group-1]         520: 	      std::forward<_Args>(__args)...); // might throw
[move_group-1]         521: 	}
[move_group-1] #13   Source "/usr/include/c++/11/bits/alloc_traits.h", line 516, in construct<planning_pipeline::PlanningPipeline, const std::shared_ptr<const moveit::core::RobotModel>&, const std::shared_ptr<rclcpp::Node>&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > [0x7f59c2fe8363]
[move_group-1]         513: 	noexcept(std::is_nothrow_constructible<_Up, _Args...>::value)
[move_group-1]         514: 	{
[move_group-1]         515: #if __cplusplus <= 201703L
[move_group-1]       > 516: 	  __a.construct(__p, std::forward<_Args>(__args)...);
[move_group-1]         517: #else
[move_group-1]         518: 	  std::construct_at(__p, std::forward<_Args>(__args)...);
[move_group-1]         519: #endif
[move_group-1] #12   Source "/usr/include/c++/11/ext/new_allocator.h", line 162, in construct<planning_pipeline::PlanningPipeline, const std::shared_ptr<const moveit::core::RobotModel>&, const std::shared_ptr<rclcpp::Node>&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > [0x7f59c2fe868c]
[move_group-1]         159: 	void
[move_group-1]         160: 	construct(_Up* __p, _Args&&... __args)
[move_group-1]         161: 	noexcept(std::is_nothrow_constructible<_Up, _Args...>::value)
[move_group-1]       > 162: 	{ ::new((void *)__p) _Up(std::forward<_Args>(__args)...); }
[move_group-1]         163: 
[move_group-1]         164:       template<typename _Up>
[move_group-1]         165: 	void
[move_group-1] #11   Source "/home/ortho/ws_moveit/src/moveit2/moveit_ros/planning/planning_pipeline/src/planning_pipeline.cpp", line 92, in PlanningPipeline [0x7f59c19864c4]
[move_group-1]          89:   , robot_model_(model)
[move_group-1]          90:   , logger_(moveit::getLogger("planning_pipeline"))
[move_group-1]          91: {
[move_group-1]       >  92:   auto param_listener = planning_pipeline_parameters::ParamListener(node, parameter_namespace);
[move_group-1]          93:   pipeline_parameters_ = param_listener.get_params();
[move_group-1]          94: 
[move_group-1]          95:   configure();
[move_group-1] #10   Source "/home/ortho/ws_moveit/build/moveit_ros_planning/planning_pipeline/planning_pipeline_parameters/include/planning_pipeline_parameters.hpp", line 83, in ParamListener [0x7f59c1991d1d]
[move_group-1]          80:   public:
[move_group-1]          81:     // throws rclcpp::exceptions::InvalidParameterValueException on initialization if invalid parameter are loaded
[move_group-1]          82:     ParamListener(rclcpp::Node::SharedPtr node, std::string const& prefix = "")
[move_group-1]       >  83:     : ParamListener(node->get_node_parameters_interface(), node->get_logger(), prefix) {}
[move_group-1]          84: 
[move_group-1]          85:     ParamListener(rclcpp_lifecycle::LifecycleNode::SharedPtr node, std::string const& prefix = "")
[move_group-1]          86:     : ParamListener(node->get_node_parameters_interface(), node->get_logger(), prefix) {}
[move_group-1] #9    Source "/home/ortho/ws_moveit/build/moveit_ros_planning/planning_pipeline/planning_pipeline_parameters/include/planning_pipeline_parameters.hpp", line 103, in ParamListener [0x7f59c199247d]
[move_group-1]         100:       }
[move_group-1]         101: 
[move_group-1]         102:       parameters_interface_ = parameters_interface;
[move_group-1]       > 103:       declare_params();
[move_group-1]         104:       auto update_param_cb = [this](const std::vector<rclcpp::Parameter> &parameters){return this->update(parameters);};
[move_group-1]         105:       handle_ = parameters_interface_->add_on_set_parameters_callback(update_param_cb);
[move_group-1]         106:       clock_ = rclcpp::Clock();
[move_group-1] #8    Source "/home/ortho/ws_moveit/build/moveit_ros_planning/planning_pipeline/planning_pipeline_parameters/include/planning_pipeline_parameters.hpp", line 201, in declare_params [0x7f59c19945f6]
[move_group-1]         198:       updated_params.planning_plugins = param.as_string_array();
[move_group-1]         199:       param = parameters_interface_->get_parameter(prefix_ + "request_adapters");
[move_group-1]         200:       RCLCPP_DEBUG_STREAM(logger_, param.get_name() << ": " << param.get_type_name() << " = " << param.value_to_string());
[move_group-1]       > 201:       updated_params.request_adapters = param.as_string_array();
[move_group-1]         202:       param = parameters_interface_->get_parameter(prefix_ + "response_adapters");
[move_group-1]         203:       RCLCPP_DEBUG_STREAM(logger_, param.get_name() << ": " << param.get_type_name() << " = " << param.value_to_string());
[move_group-1]         204:       updated_params.response_adapters = param.as_string_array();
[move_group-1] #7    Object "/opt/ros/iron/lib/librclcpp.so", at 0x7f59c30db405, in 
[move_group-1] #6    Object "/usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.30", at 0x7f59c2cae4d7, in __cxa_throw
[move_group-1] #5    Object "/usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.30", at 0x7f59c2cae276, in std::terminate()
[move_group-1] #4    Object "/usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.30", at 0x7f59c2cae20b, in 
[move_group-1] #3    Object "/usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.30", at 0x7f59c2ca2b9d, in 
[move_group-1] #2    Source "./stdlib/abort.c", line 79, in abort [0x7f59c28287f2]
[move_group-1] #1    Source "../sysdeps/posix/raise.c", line 26, in raise [0x7f59c2842475]
[move_group-1] #0  | Source "./nptl/pthread_kill.c", line 89, in __pthread_kill_internal
[move_group-1]     | Source "./nptl/pthread_kill.c", line 78, in __pthread_kill_implementation
[move_group-1]       Source "./nptl/pthread_kill.c", line 44, in __pthread_kill [0x7f59c28969fc]
[move_group-1] Aborted (Signal sent by tkill() 844762 1000)
[rviz2-2] [INFO] [1704942526.255591294] [rviz2]: Stereo is NOT SUPPORTED
[rviz2-2] [INFO] [1704942526.255646754] [rviz2]: OpenGl version: 4.6 (GLSL 4.6)
[ERROR] [move_group-1]: process has died [pid 844762, exit code -6, cmd '/home/ortho/ws_moveit/install/moveit_ros_move_group/lib/moveit_ros_move_group/move_group --ros-args --params-file /tmp/launch_params_k51bj29h --params-file /tmp/launch_params_e6n77hmh --params-file /home/ortho/ws_moveit/install/ur_moveit_config/share/ur_moveit_config/config/kinematics.yaml --params-file /tmp/launch_params_g9_tqir5 --params-file /tmp/launch_params_3ndtq39e --params-file /tmp/launch_params__ryou84b --params-file /tmp/launch_params_fz_aa0qi --params-file /tmp/launch_params_xn7kbx20 --params-file /tmp/launch_params_olp8_9zy --params-file /tmp/launch_params_n33eymhs'].
```

--- a/ur_moveit_config/launch/ur_moveit.launch.py
+++ b/ur_moveit_config/launch/ur_moveit.launch.py
@@ -156,9 +156,18 @@ def launch_setup(context, *args, **kwargs):
     # Planning Configuration
     ompl_planning_pipeline_config = {
         "move_group": {
-            "planning_plugin": "ompl_interface/OMPLPlanner",
-            "request_adapters": """default_planner_request_adapters/AddTimeOptimalParameterization default_planner_request_adapters/FixWorkspaceBounds default_planner_request_adapters/FixStartStateBounds default_planner_request_adapters/FixStartStateCollision default_planner_request_adapters/FixStartStatePathConstraints""",
-            "start_state_max_bounds_error": 0.1,
+            "planning_plugins": ["ompl_interface/OMPLPlanner"],
+            "request_adapters": [
+                "default_planning_request_adapters/ResolveConstraintFrames",
+                "default_planning_request_adapters/ValidateWorkspaceBounds",
+                "default_planning_request_adapters/CheckStartStateBounds",
+                "default_planning_request_adapters/CheckStartStateCollision",
+            ],
+            "response_adapters": [
+                "default_planning_response_adapters/AddTimeOptimalParameterization",
+                "default_planning_response_adapters/ValidateSolution",
+                "default_planning_response_adapters/DisplayMotionPath",
+            ],
         }
     }
     ompl_planning_yaml = load_yaml("ur_moveit_config", "config/ompl_planning.yaml")

